takeshape/penny
# report issue form working and other contact updates

### Test Plan (Steps to test):

1. Go to subscriptions -> orders -> report an issue
1. Type a message and submit it. 
1. Check in gorgias that the message looks as you'd expect.
2. Go to `/contact`. Notice your session info pre-populates if you're signed in.
3. Send a message. 
4. Check that it looks okay in Gorgias.
